toolchain/sem_ir/inst_namer.cpp

```diff
@@ -104,11 +104,12 @@ InstNamer::InstNamer(const File* sem_ir) : sem_ir_(sem_ir) {
   for (auto [impl_id, impl_info] : sem_ir->impls().enumerate()) {
     auto impl_scope = GetScopeFor(impl_id);
     // TODO: Provide a location for the impl for use as a disambiguator.
```

Comment:
Should this TODO be removed since it's now fingerprinted? I think the TODO 2 lines down may be more meaningful.

Reply:
Ah yeah this should go away, thanks